app/models/package_manager/base.rb

```diff
@@ -280,7 +280,7 @@ def self.deprecation_info(_name)
 
     private_class_method def self.download_async(names)
       names.each_slice(1000).each_with_index do |group, index|
-        group.each { |name| PackageManagerDownloadWorker.perform_in(index.hours, self.name.demodulize, name) }
+        group.each { |name| PackageManagerDownloadWorker.perform_in(index.hours, self, name) }
```

Comment:
Won't this try and serialize the class for the job?

Reply:
Yeah that's a good point. Maybe it should take in the full class name as a string and I'll update each spot to pass in the full name instead of assuming it will be appended with `PackageManager::`.